### PR TITLE
Adding Cloudwatch logs insights component

### PR DESCRIPTION
**Pull request by dylburger**

### CODE REVIEW

File: components/aws/aws.app.js [hunk 1]
```diff
@@ -14,5 +14,20 @@ module.exports = {
       const ses = new AWS.SES();
       return (await ses.listIdentities().promise()).Identities;
     },
+    async logsInsightsDescibeLogGroups(region, lastToken) {
+      const AWS = this.sdk(region);
+      const cloudwatchlogs = new AWS.CloudWatchLogs();
+      const data = await cloudwatchlogs
+        .describeLogGroups({ nextToken: lastToken })
+        .promise();
+      const { logGroups, nextToken } = data;
```
Comment (dylburger) [inline components/aws/aws.app.js:23]: I’ll move some of this label, value logic to the component itself

File: components/aws/cloudwatch-logs-insights/cloudwatch-logs-insights-query-results.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      console.log("No results, exiting");
+      this.db.set("startTime", now);
+    }
+
+    console.log(JSON.stringify(res, null, 2));
+
+    for (const item of res.results[0]) {
+      const { field, value } = item;
+      if (field === "@ptr") {
+        return;
```
Comment (tjk) [inline components/aws/cloudwatch-logs-insights/cloudwatch-logs-insights-query-results.js:85]: is this meant to be a return or a break?

Comment (tjk) [reply components/aws/cloudwatch-logs-insights/cloudwatch-logs-insights-query-results.js:85]: or a continue

Comment (dylburger) [reply components/aws/cloudwatch-logs-insights/cloudwatch-logs-insights-query-results.js:85]: boops, yes

**Pull request merged**